keylime/revocation_notifier.py

```diff
@@ -68,7 +70,9 @@ def stop_broker():
             os.remove(f"ipc://{_SOCKET_PATH}")
         logger.info("Stopping revocation notifier...")
         broker_proc.terminate()
-        broker_proc.join()
+        broker_proc.join(2)
```

Comment:
I am wondering if 2 seconds are appropriate. Can there be any networking involved in the shutdown process? Also, you could add a debug message when killing it so it gets noted for future troubleshooting.

Reply:
This is now 5 seconds and I added a debug message.